src/pages/hj.tsx

```diff
@@ -1,5 +1,25 @@
+import HashTag from '@/components/common/tag/HashTag';
+import StateTag from '@/components/common/tag/StateTag';
+import '@/styles/GlobalStyle';
+
 function hj() {
-  return <div>현준 테스트페이지</div>;
+  const stateTags = ['To Do', 'On Progress', 'Done'];
+  const isMobile = false;
+  const hashTag = ['프로젝트', '프론트엔드', '어려워'];
+  return (
+    <>
+      <div>
+        {stateTags.map((tag, index) => (
+          <StateTag isMobile={isMobile}>{tag}</StateTag>
+        ))}
+        {hashTag.map((tag, index) => (
+          <HashTag index={index} isMobile={isMobile}>
+            {tag}
+          </HashTag>
+        ))}
+      </div>
+    </>
+  );
```

Comment:
```suggestion
  return (
      <div>
        {stateTags.map((tag, index) => (
          <StateTag isMobile={isMobile}>{tag}</StateTag>
        ))}
        {hashTag.map((tag, index) => (
          <HashTag index={index} isMobile={isMobile}>
            {tag}
          </HashTag>
        ))}
      </div>
  );
```
[P4]: 제일 바깥의 태그가 하나의 div로만 이루어져 있으면 <></>는 없애도 될 것 같아요!